cypress/integration/hosts/host_filtering.ts

```diff
@@ -156,98 +156,121 @@ describe("Hosts page filtering from URL", () => {
   });
 });
 
-describe("Hosts page filtering from table filters", () => {
-  before(() => {
-    cy.login();
-  });
+describe(
+  "Hosts page filtering from table filters",
+  { scrollBehavior: false },
+  () => {
+    before(() => {
+      cy.login();
+    });
 
-  beforeEach(() => {
-    cy.preserveCookies();
-    cy.visit(`${hostsRoute}?limit=100&page=0`);
-    cy.dataCy("hosts-table").should("be.visible");
-    cy.dataCy("hosts-table").should("not.have.attr", "data-loading", "true");
-  });
+    beforeEach(() => {
+      cy.viewport(1920, 1080);
```

Comment:
This viewport should be set globally for the app. We shouldn't be setting different viewports on a case by case basis its better to be consistent.

Reply:
I set the viewport to 1920, 1080 in the cypress config but set it the default for the variantHistory spec where the pagination test relies on the original width